types/react-alert/index.d.ts

```diff
@@ -28,122 +29,127 @@ export interface Positions {
     BOTTOM_CENTER: 'bottom center';
     BOTTOM_RIGHT: 'bottom right';
 }
-
-export const positions: Positions;
-
-export type AlertType = 'info' | 'success' | 'error';
-
 export interface Types {
     INFO: 'info';
     SUCCESS: 'success';
     ERROR: 'error';
 }
-
-export const types: Types;
-
-export type AlertTransition = 'fade' | 'scale';
-
 export interface Transitions {
     FADE: 'fade';
     SCALE: 'scale';
 }
 
+export const positions: Positions;
+export const types: Types;
 export const transitions: Transitions;
 
-export interface AlertProviderProps extends React.HTMLAttributes<HTMLDivElement> {
+export interface AlertOptions {
     /**
-     * The margin of each alert
+     * The margin of each alert.
      *
-     * Default value: '10px'
+     * Default: '10px'
      */
-    offset?: string | undefined;
+    offset?: string;
```

Comment:
here and in other places, those were added by DT maintainer to provide better interop with TS projects using exact optional property setting:
https://www.typescriptlang.org/tsconfig#exactOptionalPropertyTypes
Those are optional in props:
https://github.com/schiehll/react-alert/blob/master/src/Provider.js#L176
this is optional

Reply:
Thanks for the reminder. I recognize the option now and will change these back later.

Reply:
Fixed in d038e29.